src/config/section/infra/primaryStorages.js

```diff
@@ -63,6 +63,24 @@ export default {
       dataView: true,
       args: ['name', 'tags', 'capacitybytes', 'capacityiops']
     },
+    {
+      api: 'updateStoragePool',
+      icon: 'pause-circle',
+      label: 'label.disable.storage',
+      message: 'message.confirm.disable.storage',
+      dataView: true,
+      defaultArgs: { enabled: false },
+      show: (record) => { return ['Up', 'Connecting', 'Down', 'ErrorInMaintenance'].includes(record.state) }
```

Comment:
@GabrielBrascher Does the backend allow disabling a pool if its state is not `Up` ?
https://github.com/apache/cloudstack/blob/master/server/src/main/java/com/cloud/storage/StorageManagerImpl.java#L782

Reply:
I think that it makes sence to disable on other cases as well. But currently the backend supports **only** if the pool is Up, I will update this on a couple of minutes.